### DISCUSSION TITLE
トピック話し合い

### DISCUSSION THREAD

**Discussion by ayame113** [22w2076e/SoftwareEcosystem-Group2#2] [General]
テストテスト

**Comment by recofita** [comment]
githubのapiを使ってデータを取る

**Comment by recofita** [comment]
https://www.nobelprize.org/about/developer-zone-2/
https://www.nobelprize.org/about/terms-of-use-for-api-nobelprize-org-and-data-nobelprize-org/

**Comment by ayame113** [comment]
12/14時点

以下のどちらかに絞った。

- ブラウザで使われている言語の仕様書（HTMLの仕様書やJavaScriptの仕様書）が誰によって維持されているのか？
    （GoogleやAppleなどのブラウザベンダーの影響力がどれくらいあるか）
- GitHubの開発に参加している人の属性（国籍とか）について調査する

**Comment by ayame113** [comment]
@shimizu-1111

**Comment by 22w2076e** [comment]
特定のプロジェクトに対する開発者の属性

**Comment by ayame113** [comment]
https://github.com/WICG
https://github.com/whatwg
https://github.com/tc39
https://github.com/wintercg

**Comment by ayame113** [comment]
トピック：特定のプロジェクトに対する開発者の属性

RQ1：特定のプロジェクトに参加している開発者の属性はどのようになっているか？（仮）
RQ2：特定のプロジェクトに参加している開発者の影響力はどのくらいあるか？（仮）

以下のことを調べる

- プロジェクトに対する開発者ごとの貢献の割合→@recofita
- 所属（`company`）→@22w2076e
- 所在地（`location`）→@ayame113
- ツイッターやってるかどうか（`twitter_username`）→余力があれば
- リポジトリ数（`public_repos`）→@shimizu-1111
- フォロー数フォロワー数（`followers` / `following`）→@shimizu-1111

**Comment by recofita** [comment]
https://github.com/o2sh/onefetch
https://github.com/o2sh/onefetch/wiki/command-line-options

**Comment by ayame113** [comment]
https://api.github.com/repos/denoland/deno/contributors?page=1&per_page=100 にアクセス（ページ番号1から順番に）（リポジトリ名は正しいものに差し替える）
↓
urlプロパティの値（`https://api.github.com/users/ry`など）にアクセスしてデータ収集

（`contributions`プロパティの値から貢献数が分かる）

複数リポジトリからとるとユーザー重複しそうなので、重複削除する

**Comment by ayame113** [comment]
スライドのURL→https://docs.google.com/presentation/d/1BVqASkmCMx3EdzWYk-KHmayu-4v2LgYnWy8C1TcD4uE/edit?usp=sharing
